src/metaUtils.cxx

```diff
@@ -36,15 +37,18 @@ inline bool IsBlank(int c)
 #include <fcntl.h>
 
 #if defined(_WIN32) && !defined(__CYGWIN__)
+#define NOMINMAX
```

Comment:
Could you add a comment indicating why this is needed. Something like what was reported in [https://github.com/facebook/folly/pull/596](https://github.com/facebook/folly/pull/596)